indico/modules/rb/util.py

```diff
@@ -321,3 +321,9 @@ def get_booking_params_for_event(event):
             'params': params,
             'time_info': time_info
         }
+
+
+def get_prebooking_collisions(reservation):
```

Comment:
unit test would probably be nice for it (that way we can also guarantee that we don't end up getting already-cancelled occurrences ;))